### ISSUE TITLE
java.lang.IllegalArgumentException: value already present

### ISSUE THREAD

**Issue by codeseedr** [basepom/duplicate-finder-maven-plugin#35]
I just hit an issue with two dependencies that try to declare jdk tools jar as a dependency. Both do it a bit differently because they use different coordinates: one (org.jolokia:jolokia-jvm:jar:agent:1.5.0) -- com.sun:tools:1.8, and the other (org.neo4j:neo4j-dbms:3.4.11) -- jdk.tools:jdk.tools:1.6. As you probably imagine, they're both scope=system deps with systemPath=${java.home}/../lib/tools.jar. 

In the above described environment plugin execution fails with:

```
java.lang.IllegalArgumentException: value already present: /Library/Java/JavaVirtualMachines/jdk1.8.0_181.jdk/Contents/Home/jre/../lib/tools.jar
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:238)
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:215)
	at org.basepom.mojo.duplicatefinder.artifact.ArtifactFileResolver.<init>(ArtifactFileResolver.java:91)
	at org.basepom.mojo.duplicatefinder.DuplicateFinderMojo.execute(DuplicateFinderMojo.java:362)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:101)
	... 20 more
```

**Comment by hgschmie** [comment]
Thanks, I will investigate this.

**Comment by hgschmie** [comment]
I have a basic fix for this; there are some challenges with the system classpath that I need to work through. I should be able to cut a new release some time this week.

**Comment by codeseedr** [comment]
Sounds good, thanks!

**Comment by hgschmie** [comment]
resolved in accf1958a969f787ec3eea7af46e64efb0d96120. Thank you for reporting this problem.
